src/plugins/data/server/search/routes.ts

```diff
@@ -39,7 +39,11 @@ export function registerSearchRoute(router: IRouter): void {
       const signal = getRequestAbortedSignal(request.events.aborted$);
 
       try {
-        const response = await context.search!.search(searchRequest, { signal }, strategy);
+        const response = await context.search!.search(
+          searchRequest,
+          { signal, rawRequest: request },
+          strategy
```

Comment:
Should strategy be a part of options?

Reply:
FWIW it would make sense as an option to me instead of its own argument (and uses the default strategy if left empty), but 🤷‍♂️